### DISCUSSION TITLE
Germ-line exomes, single samples

### DISCUSSION THREAD

**Discussion by marchoeppner** [etal/cnvkit#1023] [General]
Hi, 
apologies for using this route, but I am quite stuck with my processing and cannot find a straight answer elsewhere. 

What is the issue?
============
I have clinical exomes, typically single samples from patients across a range of diseases (routine human genetics diagnostics). For these samples, I would like to call putative CNVs using CNVkit. All exomes are sequenced on the same instrument, using the same wet-lab pipeline and exome kit (IDT xGen). Each sequencing run holds up to 48 (unrelated) samples and generates around 100X coverage per sample. Downstream processing consists of alignment (BWA) and duplicate marking against hg38 (without ALT contigs). 

1) What is the "best" reference for this setup? 
- Can I make one reference from, say, 100 previously analysed samples and re-use that?
- Should I only use the samples from one sequencing run?
- Or is a "flat" reference best?

2) BED file
The documentation is a bit unclear here. The option is called "target", but the text mentions "baits" a few times. In exome sequencing, these are two different things. "Baits" are the actual stretches used for constructing the RNA baits, whereas targets are usually the exons targetted for capture. One or more baits can map to one target (e.g. long exons). So is it baits or targets you expect here? Vedors supply BED files for both, usually. 

3) Resolution
The documentation mentions a resolution of >1MB for exomes. But the method "hmm-germline" apparently changes this? What is a realistic lower limit here?

4) Expected number of CNVs
I have no idea if my results are anywhere near realistc. Depending on my exact approach, I get anything between a few dozen (reference from all samples in a sequencing run) to up to 1000 CNV calls (flat reference). What is a "typical" number for exomes from non-tumor samples?

Thanks for the help!
/Marc

**Comment by etal** [comment]
1. Pooled is best, since the sample prep is consistent. They don't have to be matched pairs; the purpose of pooling is to reveal technical artifacts. Usually 20-30 samples is enough.
2. The command names "target" and "antitarget" are legacy quirks; you can trust the docs that these are supposed to be baited region coordinates.
3. The limit of detection really depends on the details of your samples, prep, and sequencing depth. In the data this manifests as bin size and noise. Segmentation with HMM requires at least 2 consecutive bins supporting a call, and CBS usually won't output a changepoint without 3 or 4 supporting bins. But those are technical limits, and the limits you see in analytical validation will be more conservative than that.
4. Number of segments also depends on samples, prep, and sequencing depth. A few dozen is reasonable and typical for a comprehensive cancer gene panel, maybe a few hundred for a whole exome, depending on the cancer type. For non-cancer constitutional samples, that 1MB limit makes all the difference -- a normal genome has lots of smallish structural variants and STRs, but if you're investigating a rare disease case then you'd tend to see maybe one gene-sized or larger deletion that is real and causative.
